README.md

```diff
@@ -1,3 +1,4 @@
+
```

Comment:
It seems that this extra empty line is not needed here. Can you pls remove it?

Reply:
Sure thing, @deniskaber!